### PR TITLE
Add pxd volume options inline in the name

### PR DESCRIPTION
**Pull request by disrani-px**

### CODE REVIEW

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java [hunk 1]
```diff
@@ -705,8 +707,15 @@ private static void addDockerVolumes(Protos.ContainerInfo.Builder containerInfo,
                 driverOptions.addAllParameter(paramsList);
                 driverOptions.build();
 
-                LOGGER.info("Adding DOCKER volume {} with driver {} to pod", volumeName,
-                        dockerVolume.getDriverName());
+                // For pxd also add params inline
+                if (dockerVolume.getDriverName().equals("pxd")) {
+                    volumeName = "name=" + volumeName;
```
Comment (piyush-nimbalkar) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java]: Does the volume name have the suffixed index here already?

Comment (disrani-px) [reply sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java]: Yes, it is done above on line 657

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java [hunk 2]
```diff
@@ -705,8 +707,15 @@ private static void addDockerVolumes(Protos.ContainerInfo.Builder containerInfo,
...
-                LOGGER.info("Adding DOCKER volume {} with driver {} to pod", volumeName,
-                        dockerVolume.getDriverName());
+                // For pxd also add params inline
+                if (dockerVolume.getDriverName().equals("pxd") && dockerVolume.getDriverOptions() != null) {
+                    StringBuffer nameBuf = new StringBuffer();
+                    nameBuf.append("name=" + volumeName);
+                    for (Map.Entry<String, String> option : dockerVolume.getDriverOptions().entrySet()) {
+                        nameBuf.append(";" + option.getKey() + "=" + option.getValue());
+                    }
+                    volumeName = nameBuf.toString();
```
Comment (piyush-nimbalkar) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java:717]: We should set the driverOptions to empty string once we have added them to the name. Not sure what happens when you pass as both inline spec and options.

**Review by piyush-nimbalkar — APPROVED** [review]

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java (hunk 2 above)
Comment (disrani-px) [reply sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java:717]: I've let that be to keep the old behavior the same, it should not cause any issues

**Pull request merged**